### DISCUSSION TITLE
Fcitx-rime 当前在全键盘下能否实现类似于 Gboard 的“瞎打”效果

### DISCUSSION THREAD

**Discussion by dongfengweixiao** [fcitx5-android/fcitx5-android#491] [General | 讨论]
因为之前长期使用的是 Gboard 全键盘，习惯了“瞎打”，就是在使用输入法进行输入时，每一个按键不需要按得特别准确，按在附近得按键上，都能比较准确得识别出来。
在这点上，国内得搜狗，百度之类得输入法都无法做到这样的精准度， Fcitx 有办法能做到吗？

**Comment by dongfengweixiao** [comment]
以“何以当归”为例：

在使用 gboard 输入时，当第一次输入为“heyisanggui”，无法联想到正确词组：

![微信图片_20240429101414](https://github.com/fcitx5-android/fcitx5-android/assets/1300822/ee86ae5c-9d86-4725-8966-b6c8a52bd957)

在gboard面板将“sang”调整为“dang”：
![微信图片_20240429101435](https://github.com/fcitx5-android/fcitx5-android/assets/1300822/7639d225-f73c-45b5-90ee-4fda389182e5)

再次输入错误的内容 “heyisangguu”：
![微信图片_20240429101442](https://github.com/fcitx5-android/fcitx5-android/assets/1300822/145a6cc9-a885-4333-8ced-d8c3eea4fbc2)

**Comment by rocka** [comment]
我猜你想要的是拼音按键纠错。 rime 确实可以实现纠错功能，但实际效果如何取决于如何编写配置。我不用 rime 所以不做评价。

自带（fcitx5-chinese-addons）的拼音也有一点邻键纠错的能力，在拼音设置最下面，将“纠错布局”设置为 QWERTY 即可启用。如果按错的只是同一行的按键，还是有较大的概率能够纠正过来的。

![Screenshot_20240429-103029](https://github.com/fcitx5-android/fcitx5-android/assets/13914967/78d496d9-e907-49a5-b862-d1699345c07c)

**Reply by dongfengweixiao** [reply]
好的，我去看看rime在这方面如何配置。
gboard在纠错这方面功能的确是蛮好的，当前验证能实现在十字方向上的纠错。

**Comment by dongfengweixiao** [comment]
确认了下，当使用ice rime时，使用translator/enable_correction会导致输入法无法输入中文，在ice rime发issue咨询了：https://github.com/iDvel/rime-ice/issues/836

**Reply by expoli** [reply]
这个功能目前来说在打单字的时候确实不好用，会造成很多重码。

你看一下最近的issue，好像是rime那边有对应的bug，不知道有没有修好

https://github.com/fcitx5-android/fcitx5-android/issues/483

**Reply by dongfengweixiao** [reply]
https://github.com/iDvel/rime-ice/issues/836

rime-ice那边回复：是librime版本的bug，需要升级或者是降级版本。
